js/packages/snippets/src/browser/getChildFramesInfo.js

```diff
@@ -1,10 +1,11 @@
-function getChildFramesInfo() {
-  const frames = document.querySelectorAll('frame, iframe')
-  return Array.prototype.map.call(frames, frameElement => [
-    frameElement,
-    !frameElement.contentDocument,
-    frameElement.src,
-  ])
+function getChildFramesInfo(root) {
+  root = root || document
```

Comment:
@zamboney Pay attention to the convention among all snippets, it always accepts only one (or none) argument and this argument is an array. So I recommend this

```suggestion
function getChildFramesInfo([root = document] = []) {
```

Reply:
I try your suggestion but when I do I get a `null` as an argument and it's over writing the default value, I will give it a try tomorrow...

Reply:
This is how it works everywhere else (you can check yourself), so I will be surprised if this isn't going to work

Reply:
@kyrylodolynskyi, your suggestion didn't work because the parameter that is supplied by the driver is `null`, but I made some changes by moving the recursive function inside. I hope this better meets your needs